libs/isograph-react/src/loadable-hooks/usePagination.ts

```diff
@@ -0,0 +1,315 @@
+import { ItemCleanupPair } from '@isograph/disposable-types';
```

Comment:
hm... I'm thinking this might be more accurate to call this useConnectionSpecPagination? Is that too verbose?